### PR TITLE
Add confirmation workflow for commands

### LINKED ISSUE #105: Confirmation for clear command
For a command with this much consequence, it should have a confirmation before someone accidentally nukes his whole contact list

### PR DESCRIPTION
**Pull request by mongj**
Closes #105 

Adds a new confirmation workflow for commands (currently applies to clear, delete, and deletelog)

### CODE REVIEW

File: src/main/java/careconnect/logic/commands/CommandStack.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package careconnect.logic.commands;
```
Comment (SuhailLoya) [inline src/main/java/careconnect/logic/commands/CommandStack.java:1]: v cool!!

**Review by SuhailLoya — APPROVED** [review]
just need fix the java ci fails, apart from that LGTM!! v cool!!!!

File: src/main/java/careconnect/logic/commands/ClearCommand.java [hunk 2]
```diff
@@ -13,10 +13,23 @@ public class ClearCommand extends Command {
     public static final String COMMAND_WORD = "clear";
     public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
 
+    public ClearCommand() {
+        super(true);
```
Comment (celeschai) [inline src/main/java/careconnect/logic/commands/ClearCommand.java:17]: Good use of inheritance!

File: src/test/java/careconnect/logic/commands/DeleteCommandTest.java
Comment (celeschai) [inline src/test/java/careconnect/logic/commands/DeleteCommandTest.java:1]: Good job on keeping up with testing!

**Comment by SuhailLoya** [comment]
thanks!!!

**Review by celeschai — APPROVED** [review]
LGTM!

**Pull request merged**